prometheus-exporters-formula/prometheus-exporters/map.jinja

```diff
@@ -9,6 +9,9 @@
         'postgres_exporter_package': 'golang-github-wrouesnel-postgres_exporter',
         'postgres_exporter_service': 'prometheus-postgres_exporter',
         'postgres_exporter_service_config': '/etc/sysconfig/prometheus-postgres_exporter',
+        'exporter_exporter_package': 'golang-github-QubitProducts-exporter_exporter',
+        'exporter_exporter_service': 'prometheus-exporter_exporter',
+        'exporter_exporter_service_config': '/etc/sysconfig/prometheus-exporter_exporter',
```

Comment:
Is the plan to package, ship and support `exporter_exporter` only for SUSE distributions? Otherwise we would of course need to add it below to the other sections respectively. The respective config files seem to be included with this patch at least (see above).